app/controllers/security_group_controller.rb

```diff
@@ -262,9 +261,7 @@ def rule_changed?(params)
   end
 
   def changed?(param, field)
-    if param && !param.empty?
-      return true if field != param
-    end
+    return true if param.present? && field != param
```

Comment:
I'd say you can just remove the `return true if `

Reply:
OK, changed as suggested.